nemo/collections/asr/losses.py

```diff
@@ -58,3 +59,9 @@ def _loss(self, log_probs, targets, input_length, target_length):
 
     def _loss_function(self, **kwargs):
         return self._loss(*(kwargs.values()))
+
+
+class CrossEntropyLossNM(losses.CrossEntropyLoss):
```

Comment:
We already have crossentropyloss defined elsewhere, please use that

Reply:
Good point, added alias in asr __init__.py to get the backend function. Added "NM" to it so that it is clear that this is a NM and not pytorch loss.